cms/static/js/spec_helpers/modal_helpers.js

```diff
@@ -32,6 +32,13 @@ define(["jquery", "common/js/spec_helpers/template_helpers", "js/spec_helpers/vi
             return modalElement.length > 0;
         };
 
+        isModalFocused = function(modal) {
+            var modalElement = getModalElement(modal),
+                modalFocusWindow = modalElement.find('.modal-window');
+
+            return modalFocusWindow.focus();
```

Comment:
As we discussed, you don't want to actually call focus in the test method-- you want to verify if focus was called on the modal as a result of show being called.